cvat-ui/src/components/analytics-page/analytics-performance.tsx

```diff
@@ -0,0 +1,222 @@
+    'rgba(255, 99, 132, 0.5)',
+    'rgba(53, 162, 235, 0.5)',
+    'rgba(170, 83, 85, 0.5)',
+    'rgba(44, 70, 94, 0.5)',
+    'rgba(28, 66, 98, 0.5)',
+];
+
+function AnalyticsOverview(props: Props): JSX.Element | null {
+    const { report, onTimePeriodChange } = props;
+
+    // TODO make it more expressive
+    if (!report) return null;
+    const layout: any = [];
+    let histogramCount = 0;
+    let numericCount = 0;
+    const views = Object.entries(report.statistics).map(([name, entry]) => {
+        switch (entry.defaultView) {
+            case AnalyticsEntryViewType.NUMERIC: {
+                layout.push({
+                    i: name,
+                    w: 2,
+                    h: 1,
+                    x: 2,
+                    y: numericCount,
+                });
+                numericCount += 1;
+                const { value } = entry.dataseries[Object.keys(entry.dataseries)[0]][0];
+                return ({
+                    view: (
+                        <AnalyticsCard
+                            title={entry.title}
+                            value={typeof value === 'number' ? value.toFixed(2) : 0}
+                            bottomElement={<Text>{entry.description}</Text>}
+                            key={name}
+                        />
+                    ),
+                    key: name,
+                });
+            }
+            case AnalyticsEntryViewType.HISTOGRAM: {
+                const firstDataset = Object.keys(entry.dataseries)[0];
+                const dateLabels = entry.dataseries[firstDataset].map((dataEntry) => (
+                    moment.utc(dataEntry.date).local().format('YYYY-MM-DD')
+                ));
+
+                const { dataseries } = entry;
+                entry.transformations.forEach((transform) => {
+                    if (transform.binary) {
+                        let operator: (left: number, right: number) => number;
+                        switch (transform.binary.operator) {
+                            case '+': {
+                                operator = (left: number, right: number) => left + right;
+                                break;
+                            }
+                            case '-': {
+                                operator = (left: number, right: number) => left - right;
+                                break;
+                            }
+                            case '*': {
+                                operator = (left: number, right: number) => left * right;
+                                break;
+                            }
+                            case '/': {
+                                operator = (left: number, right: number) => (right !== 0 ? left / right : 0);
+                                break;
+                            }
+                            default: {
+                                throw Error(`Operator type ${transform.binary.operator} is not supported`);
```

Comment:
```suggestion
                                throw new Error(`Operator type ${transform.binary.operator} is not supported`);
```

The same, should we fail the whole UI in case of not supported operator?